src/translations/home.js

```diff
@@ -146,11 +146,11 @@ export default {
             nl: "APIs"
         },
         desc: {
-            en: "Our API references, with endpoints, parameters you can use and consoles to run interactive queries",
-            fr: "Nos références d'API, avec les points terminaux, les paramètres que vous pouvez utiliser et les consoles pour exécuter des requêtes interactives",
-            es: "Nuestra referencia de API, con extremos, parámetros que se pueden utilizar y consolas para ejecutar consultas interactivas",
-            de: "Unsere API-Referenzen mit Endpunkten, Parametern, die Sie verwenden können, und Konsolen zum Ausführen interaktiver Abfragen",
-            nl: "Onze API-referenties met eindpunten, parameters die u kunt gebruiken en consoles om interactieve onderzoeken te voeren"
+            en: "Our API references, with endpoints, parameters you can use and consoles to run interactive queries.",
+            fr: "Nos références d'API, avec les points terminaux, les paramètres que vous pouvez utiliser et les consoles pour exécuter des requêtes interactives.",
```

Comment:
"les consoles pour exécuter des requêtes interactives." 
That doesn't seem to be true